language-plutus-core/src/Language/PlutusCore/TH.hs

```diff
@@ -6,7 +6,7 @@
 
 module Language.PlutusCore.TH (plcTerm, plcType, plcProgram) where
 
-import           Language.Haskell.TH        hiding (Name, Type)
+import           Language.Haskell.TH           hiding (Name, Type)
```

Comment:
I thought we agreed not to make random formatting changes to unrelated files?

Reply:
This is `stylish-haskell`.

Reply:
Yes, but we haven't agreed to use that everywhere yet, and changes like that should happen all at once. Doing it this way just maximizes the amount of conflicts.